packages/datadog-instrumentations/src/child_process.js

```diff
@@ -17,7 +17,7 @@ const execAsyncMethods = ['execFile', 'spawn']
 const names = ['child_process', 'node:child_process']
 
 // child_process and node:child_process returns the same object instance, we only want to add hooks once
-let patched = false
+const patched = new WeakSet()
```

Comment:
addHook could be executed twice with different object as `childProcess`, in that case, we want to instrument both.

```
import { method } from 'child-process'
// and
import childProcess from 'child-process'
```